WcaOnRails/app/views/competitions/_index_table.html.erb

```diff
@@ -17,7 +17,7 @@
         <% else %>
           <%= icon('hourglass-start', title: "Starts in #{ pluralize((competition.start_date - Date.today).to_i, "day") }", data: { toggle: "tooltip" }) %>
         <% end %>
-        <%= wca_date_range(competition.start_date, competition.end_date)  %>
+        <%= competition.start_date && competition.end_date ? wca_date_range(competition.start_date, competition.end_date) : "Unscheduled" %>
```

Comment:
We definitely don't want such a `"Unscheduled"` competitions to be shown on the competitions index. I think they aren't placed in the `@competitions`.